_build/build.transport.php

```diff
@@ -23,7 +23,7 @@ function getSnippetContent($filename = '') {
     /* define version */
     define('PKG_NAME', 'Commerce_Cursus');
     define('PKG_NAMESPACE', 'commerce_cursus');
-    define('PKG_VERSION', '0.1.0');
+    define('PKG_VERSION', '1.1.1');
```

Comment:
In general, please don't bump versions in PRs. We may release it as a different number or end up bundle changes together.